sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -18,22 +22,66 @@ You may also refer to [Authenticating a user account interactively in the browse
+| ------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [AzureCliCredential](#authenticating-a-user-account-with-azure-cli)                  | Authenticate in a development environment with the Azure CLI.                                                                                                                                                                                                                                                           | [Install Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) and [login using az cli command](https://docs.microsoft.com/cli/azure/authenticate-azure-cli) |
+| [VisualStudioCodeCredential](#authenticating-a-user-account-with-visual-studio-code) | Authenticate in a development environment with Visual Studio Code.                                                                                                                                                                                                                                                      | Install [VS Code](https://code.visualstudio.com/), [VS Code Azure extension](https://code.visualstudio.com/docs/azure/extensions) and login using the extension.          |
+| [DefaultAzureCredential](#authenticating-with-defaultazurecredential)                | Tries `AzureCliCredential`, `VisualStudioCodeCredential`, and other credentials sequentially until one of them succeeds. Use this to have your application authenticate using developer tools, service principals or managed identity based on what is available in the current environment without changing your code. |
+
+### Authenticating Service Principals
+
+An Azure service principal is an identity created for use with applications, hosted services, and automated tools to access Azure resources. This access is restricted by the roles assigned to the service principal, giving you control over which resources can be accessed and at which level. For security reasons, it's always recommended to use service principals with automated tools rather than allowing them to log in with a user identity.
+
+To learn more, read [Application and service principal objects in Azure Active Directory][app-register-service-principal]
+
+Set up:
```

Comment:
Nit: are we sure it's "Set up" and not "Setup"?

Reply:
They convey different things: https://writingexplained.org/setup-vs-set-up-difference

One main difference is that `setup` can be a noun. "the setup". I'll leave it up to you.

Reply:
Updated to "setup" in https://github.com/Azure/azure-sdk-for-js/pull/14920/commits/f8a32bbd652ef0492831f07fadf876c2016b7a97